lstchain/io/io.py

```diff
@@ -332,15 +337,33 @@ def auto_merge_h5files(
     else:
         keys = set(nodes_keys)
 
+    copy_keys = {}
```

Comment:
As commented below, I think the default should be the keys we know should be copied instead of merged (monitoring and instrument for starters)

Reply:
I added it in the tool